src/pynxtools/nomad/parser.py

```diff
@@ -383,13 +385,21 @@ def _get_chemical_formulas(self) -> Set[str]:
             if sample.get("atom_types__field") is not None:
                 atom_types = sample.atom_types__field
                 if isinstance(atom_types, list):
-                    atomlist = atom_types
+                    for symbol in atom_types:
+                        if symbol in chemical_symbols[1::]:
```

Comment:
I think it would also make sense to emit a warning here that an element was ignored during parsing

Reply:
Agree, implemented